OpenFAST/IEA-15-240-RWT-Monopile/IEA-15-240-RWT-Monopile_SubDyn.dat

```diff
@@ -1,136 +1,135 @@
+     2      0.00000     0.00000    -29.999           1          0.0         0.0         0.0        0.0
+     3      0.00000     0.00000    -25.0000          1          0.0         0.0         0.0        0.0
+     4      0.00000     0.00000    -24.999           1          0.0         0.0         0.0        0.0
+     5      0.00000     0.00000    -20.0000          1          0.0         0.0         0.0        0.0
+     6      0.00000     0.00000    -19.999           1          0.0         0.0         0.0        0.0
+     7      0.00000     0.00000    -15.0000          1          0.0         0.0         0.0        0.0
+     8      0.00000     0.00000    -14.999           1          0.0         0.0         0.0        0.0
+     9      0.00000     0.00000    -10.0000          1          0.0         0.0         0.0        0.0
+    10      0.00000     0.00000     -9.999           1          0.0         0.0         0.0        0.0
+    11      0.00000     0.00000     -5.0000          1          0.0         0.0         0.0        0.0
+    12      0.00000     0.00000     -4.999           1          0.0         0.0         0.0        0.0
+    13      0.00000     0.00000      0.0000          1          0.0         0.0         0.0        0.0
+    14      0.00000     0.00000      0.001           1          0.0         0.0         0.0        0.0
+    15      0.00000     0.00000      5.0000          1          0.0         0.0         0.0        0.0
+    16      0.00000     0.00000      5.001           1          0.0         0.0         0.0        0.0
+    17      0.00000     0.00000     10.0000          1          0.0         0.0         0.0        0.0
+    18      0.00000     0.00000     10.001           1          0.0         0.0         0.0        0.0
+    19      0.00000     0.00000     15.0000          1          0.0         0.0         0.0        0.0
+------------------- BASE REACTION JOINTS: 1/0 for Locked/Free DOF @ each Reaction Node ---------------------
+             1   NReact      - Number of Joints with reaction forces; be sure to remove all rigid motion DOFs of the structure  (else det([K])=[0])
+ RJointID    RctTDXss    RctTDYss    RctTDZss    RctRDXss    RctRDYss    RctRDZss     SSIfile   [Global Coordinate System]
+    (-)       (flag)      (flag)      (flag)      (flag)      (flag)      (flag)     (string)
+     1           1           1           1           1           1           1
+------- INTERFACE JOINTS: 1/0 for Locked (to the TP)/Free DOF @each Interface Joint (only Locked-to-TP implemented thus far (=rigid TP)) ---------
+             1   NInterf     - Number of interface joints locked to the Transition Piece (TP):  be sure to remove all rigid motion dofs
+IJointID   ItfTDXss    ItfTDYss    ItfTDZss    ItfRDXss    ItfRDYss    ItfRDZss     [Global Coordinate System]
+  (-)       (flag)      (flag)      (flag)      (flag)      (flag)      (flag)
+  19           1           1           1           1           1           1
```

Comment:
I think that is because my editor changed the Windows end-of-line character (CRLF) to a Unix one (LF).  This line 56 is the only one that changed.